benchmark/flink-kafka/README.md

```diff
@@ -44,34 +37,34 @@ git clone https://github.com/nexmark/nexmark.git
    warned that `build.sh` doesn't bail out on errors, so you might
    still get something there even if the build partially fails.
 
-5. Extract Nexmark archive:
+4. Extract Nexmark archive:
 
 ```
 tar xzf nexmark/nexmark-flink/nexmark-flink.tgz
 ```
 
-6. Download and unpack Kafka:
+5. Download and unpack Kafka:
 
 ```
-wget https://dlcdn.apache.org/kafka/3.4.0/kafka_2.13-3.4.0.tgz
-tar xfz kafka_2.13-3.4.0.tgz
+wget https://downloads.apache.org/kafka/3.8.0/kafka_2.13-3.8.0.tgz
+tar xfz kafka_2.13-3.8.0.tgz
 ```
 
-7. Download the Flink connector for Kafka:
+6. Download the Flink connector for Kafka:
 
 ```
-wget https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-kafka/1.17.0/flink-sql-connector-kafka-1.17.0.jar
+wget https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-kafka/3.2.0-1.19/flink-sql-connector-kafka-3.2.0-1.19.jar
 ```
 
-8. Build a Docker container based on the upstream Flink container that
+7. Build a Docker container based on the upstream Flink container that
    also contains Nexmark and Kafka, and give it the tag
    `nexmark-kafka`:
 
 ```
 docker build -t nexmark-kafka .
 ```
 
```

Comment:
Do we still need to do this now that we have the `build` section in docker-compose.yml?